websites/P/Pinterest/presence.ts

```diff
@@ -3,143 +3,137 @@ const presence = new Presence({
 	}),
 	browsingTimestamp = Math.floor(Date.now() / 1000);
 
-let user: Element | HTMLElement | string,
-	search: Element | HTMLElement | string,
-	title: Element | HTMLElement | string;
+const enum Assets {
+	Likes = "https://i.imgur.com/394dVMI.png",
+	Logo = "https://cdn.rcd.gg/PreMiD/websites/P/Pinterest/assets/logo.png",
+	Loading = `https://i.imgur.com/${"6s5f2TA"}.gif`,
```

Comment:
Alternate solution for enum
```suggestion
	Loading = "https://i.imgur.com/\u002e.gif",
```